Fix changelog and docs

matthiasschaub opened the pull request:
### Description
Fixes deafult value in configuration docs.
Fixes markdown table, missing links and duplicated heading in CHANGELOG.md

Gigaszi commented on CHANGELOG.md:
```diff
@@ -41,6 +38,7 @@
 - Make sure to rename the API query parameter `layerName` to `layerKey` and API endpoint `listLayerNames` to `listLayerKeys` ([#376])
 - To continue to retrieve the properties of the GeoJSON API response as flat list, you need to set the API request parameter `flattem` to `True` ([#375])
```
Maybe replace the 'flattem' with 'flatten' while you are at it anyway

matthiasschaub replied:
Done

Gigaszi commented:
Currently there are two captions Bug FIxes and two New Features in the changelog. Maybe this should also be fixed in this PR.

matthiasschaub commented:
> Currently there are two captions Bug FIxes and two New Features in the changelog. Maybe this should also be fixed in this PR.

Thanks for spotting. I fixed this and updated to main. Could you have another look?
